full-stack-tests/presentation/src/backend/PresentationManager.test.ts

```diff
@@ -6,7 +6,10 @@ import { expect } from "chai";
 import { ClientRequestContext, Guid } from "@bentley/bentleyjs-core";
 import { IModelDb, SnapshotDb } from "@bentley/imodeljs-backend";
 import { PresentationManager } from "@bentley/presentation-backend";
-import { ContentSpecificationTypes, Field, KeySet, PresentationUnitSystem, Ruleset, RuleTypes } from "@bentley/presentation-common";
+import {
+  ContentSpecificationTypes, DisplayValuesArray, DisplayValuesMap, KeySet, PresentationUnitSystem, Ruleset, RuleTypes,
+} from "@bentley/presentation-common";
+import { findFieldByLabel } from "../frontend/Content.test";
```

Comment:
I'd like this function moved to some directory common to backend and frontend tests. Generally it's unsafe for backend code to use frontend code and vice versa, so let's make it clear there's nothing frontend-specific in the function by moving it out of "frontend" dir.

Reply:
Moved it to common\src\presentation-common. Maybe there is a better place for it?

Reply:
I think it should still be under full-stack-tests. Maybe in `{imodeljs}/full-stack-tests/presentation/src/Utils.ts`?